my_hebrew_dates/templates/hebcal/calendar_list.html

```diff
@@ -3,44 +3,57 @@
 {% block title %}Calendars{% endblock title %}
 
 {% block content %}
-<div class="container">
-  <div class="row">
-    <div class="col-md-8 offset-md-2">
-      <h1 class="text-center">My Calendars</h1>
-      <hr>
-      {% for calendar in calendar_list %}
-        <div class="row mb-3">
-          <div class="col">
-            <div class="d-flex align-items-center justify-content-between bg-light p-3">
-              <div>
-                <h5 class="mb-0">
-                  <a href="{% url 'hebcal:calendar_share' calendar.uuid %}">{{ calendar.name }}</a>
-                </h5>
-                <p class="text-muted mb-0">{{ calendar.timezone }}</p>
-              </div>
-              <div>
-                <input hidden id="calendar_file_url{{calendar.uuid}}" value="https://{{domain_name}}{% url 'hebcal:calendar_share' calendar.uuid %}">
-                <button onclick="shareFile('{{calendar.uuid}}')" class="btn btn-primary" type="button" title="Share with friends and family">
-                  <i class="bi bi-share"></i>
-                </button>
-
-                <a href="{% url 'hebcal:calendar_edit' calendar.pk %}" class="btn btn-primary">Edit</a>
-                <a href="{% url 'hebcal:calendar_delete' calendar.pk %}" class="btn btn-danger">Delete</a>
-              </div>
-            </div>
+  <div class="container mt-5">
+    <div class="row">
+      <div class="col-md-8 offset-md-2">
+        <div class="row position-relative">
+          <h1 class="text-center col-11">My Calendars</h1>
+          <div class="col-1 align-items-center align-self-center align-content-center justify-content-center">
+            <a href="{% url 'hebcal:calendar_new' %}" class="btn btn-success">New</a>
           </div>
         </div>
-        {% endfor %}
-        <div class="text-center">
-          <a href="{% url 'hebcal:calendar_new' %}" class="btn btn-success">New</a>
-        </div>
+
+        <hr>
+        <table class="w-100 text-center">
+          <thead>
+            <tr>
+              <th class="text-center">Name</th>
+              <th class="text-center">Time Zone</th>
+              <th class="text-center"># of Events</th>
+              <th class="text-center">Actions</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for calendar in calendar_list %}
+            <tr>
+              <td class="py-3"><a class="text-decoration-none fs-6" href="{% url 'hebcal:calendar_share' calendar.uuid %}">B-day</a></td>
```

Comment:
Why `B-day`?
Do you mean the name of the calendar?
That can be accessed by `calendar.name`

Reply:
I committed the fix